packages/jest-emotion/README.md

```diff
@@ -94,14 +94,67 @@ import styled from '@emotion/styled'
 expect.extend(matchers)
 
 test('renders with correct styles', () => {
-  const H1 = styled.h1`
+  const Svg = styled('svg')`
+    width: 100%;
+  `
+
+  const Div = styled('div')`
     float: left;
+    height: 80%;
+    &:hover {
+      width: 50px;
+    }
+    ${Svg} {
+      fill: green;
+    }
+    span {
+      color: yellow;
+    }
+    @media screen and (max-width: 1200px) {
+      font-size: 14px;
+    }
   `
 
-  const tree = renderer.create(<H1>hello world</H1>).toJSON()
+  const tree = renderer
+    .create(
+      <Div>
+        <Svg />
+        <span>Test</span>
+      </Div>
+    )
+    .toJSON()
 
   expect(tree).toHaveStyleRule('float', 'left')
-  expect(tree).not.toHaveStyleRule('color', 'hotpink')
+  expect(tree).not.toHaveStyleRule('height', '100%')
+})
+```
+
+You can provide additional options for `toHaveStyleRule` matcher.  
+`target` - option helps to specify target where style rule should be found.
```

Comment:
This should mention that's it's a css selector.

Reply:
yes